### DISCUSSION TITLE
Final File Sizes

### DISCUSSION THREAD

**Discussion by sharma-TIP** [CMSgov/price-transparency-guide#507] [Polls]
It's almost time that Org's have a good idea of the final JSON file sizes. Just wanted to see how big the files are:

**Comment by Devyandu** [comment]
~200 files ~5gig each.

**Reply by tysonpkp** [reply]
Do you mind if I ask the reason for the number of files? Was this to keep the file size in a manageable level, or just how the pricing structures are broken down?

**Reply by Devyandu** [reply]
The files are at the same level as the pricing structure (for the most part)

**Reply by nmayle** [reply]
Do you mean you have 200 plans or 200 networks?  Are you using table-of-contents files?
